src/components/side-by-side/side-by-side.scss

```diff
@@ -5,6 +5,7 @@
   position: relative;
   margin: auto;
   flex: 1;
+  font-family: sans-serif;
```

Comment:
remove

Reply:
since side-by-side layouts are not inside gui-wrapper the corporative font does't applied to child components. Current PR fixes the font issue for side-by-side layouts.

Reply:
I would fix it in the ui

Reply:
font-family will be handled by additional wrapper for PlayerArea